packages/golden_toolkit/lib/src/configuration.dart

```diff
@@ -30,7 +30,7 @@ class GoldenToolkit {
   /// In most cases, this can be applied in your flutter_test_config.dart to wrap every test in its own zone
   static T runWithConfiguration<T>(
     T Function() body, {
-    @required GoldenToolkitConfiguration config,
+    GoldenToolkitConfiguration? config,
```

Comment:
I think this should be required. If no configuration is supplied, then there is no need to call "runWithConfiguration()"